docker/bitcoin-regtest/entrypoint.sh

```diff
@@ -5,10 +5,17 @@ set -e
 cd /opt/coins/nodes/bitcoin_regtest
 
 echo "Starting bitcoin regtest backend service"
-/opt/coins/nodes/bitcoin_regtest/bin/bitcoind -datadir=/opt/coins/data/bitcoin_regtest/backend -conf=/opt/coins/nodes/bitcoin_regtest/bitcoin_regtest.conf &
+bin/bitcoind -datadir=/opt/coins/data/bitcoin_regtest/backend -conf=/opt/coins/nodes/bitcoin_regtest/bitcoin_regtest.conf &
+
+sleep 3
+
+# generate test wallet and blocks
+bin/bitcoin-cli -rpcport=18021 -rpcuser=rpc -rpcpassword=rpc createwallet "tenv-wallet"
+bin/bitcoin-cli -rpcport=18021 -rpcuser=rpc -rpcpassword=rpc settxfee 0.00001
+bin/bitcoin-cli -rpcport=18021 -rpcuser=rpc -rpcpassword=rpc -generate 150
 
 # start blockbook
 cd /opt/coins/blockbook/bitcoin_regtest
 
 echo "Starting blockbook service"
-/opt/coins/blockbook/bitcoin_regtest/bin/blockbook -blockchaincfg=/opt/coins/blockbook/bitcoin_regtest/config/blockchaincfg.json -datadir=/opt/coins/data/bitcoin_regtest/blockbook/db -sync -internal=:19021 -public=:19121 -certfile=/opt/coins/blockbook/bitcoin_regtest/cert/blockbook -explorer= -log_dir=/opt/coins/blockbook/bitcoin_regtest/logs
+bin/blockbook -blockchaincfg=config/blockchaincfg.json -datadir=/opt/coins/data/bitcoin_regtest/blockbook/db -sync -internal=:19021 -public=:19121 -certfile=cert/blockbook -explorer= -log_dir=logs
```

Comment:
Maybe last thing for discussion - what about splitting these long commands into multiple lines, like
```sh
bin/blockbook \
    -blockchaincfg=config/blockchaincfg.json \
    -datadir=/opt/coins/data/bitcoin_regtest/blockbook/db \
    -sync \
    -internal=:19021 \
    -public=:19121 \
    -certfile=cert/blockbook \
    -explorer= \
    -log_dir=logs
```

Actually seeing this uncovered to me that `-explorer` is empty is it alright?

Reply:
Yes tahat should be Ok, all of the packages are run this way. Don't really think it's necessary to modify the commands any further I'm fine with how it is know but in all honesty whatever you guys like more I will be fine with.

Reply:
Yeah that looks nicer. So @vdovhanych let's do this final minor change and ship it?

Reply:
Done, will test the docker image after it's built and if it's ok i will merge it then.

Reply:
I agree with merging

Reply:
Yes, do it!